example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This file should be part of the `.gitignore`.  See [Flutter gitignore template](https://github.com/toptal/gitignore/blob/master/templates/Flutter.gitignore#L53).